packages/docs/docs/hello-world/create-react-app.md

```diff
@@ -0,0 +1,211 @@
+
+Click on "New" in the toolbar
+
+![New Patient button](/img/hello-world/new-patient-button.png)
+
+Add a patient name
+
+![Patient name](/img/hello-world/patient-name.png)
+
+Scroll to the bottom and click "OK"
+
+![OK Button](/img/hello-world/ok-button.png)
+
+Congrats, you created a patient!
+
+Go to the "Details" tab:
+
+Copy the "ID" field, because we will need it later.  Patients often have many identifiers: MRN (medical record number), SSN (social security number), drivers' license, etc.  The primary "ID" is a special ID that we will use in the Medplum API.
+
+## Find Client Application ID
+
+Now let's get our Client Application ID, because we will also need that later.  Click on the menu button in the top left corner:
+
+Click on "Client Applications":
+
+When you registered your account, Medplum automatically created a default client.  Click on that row:
+
+And copy the Client Application ID:
+
+To recap:  You registered a new account, created a patient, and should have a Patient ID and a Client Application ID.
+
+## Create a React app
+
+First, make sure you have Node JS and npm installed.  If not, follow instructions here:
+
+https://nodejs.org/en/download/
+
+Open a terminal window.  Create a new React app using create-react-app.  We're going to use the TypeScript variant:
+
+```bash
+npx create-react-app medplum-hello-world --template typescript
+cd medplum-hello-world
+npm start
+```
+
+That should open a browser window to the default React app:
+
+Go back to the terminal.  Stop the Node JS process using Ctrl+C.
+
+## Add Medplum dependencies
+
+Next let's add Medplum dependencies.
+
+npm install @medplum/core @medplum/ui
+
+The @medplum/core package includes the basic Medplum client and utilities.
+
+The @medplum/ui package includes the React components.
+
+## Add Medplum authentication
+
+Next let's use the Medplum React components.  Open the medplum-hello-world folder in your favorite editor.  
+
+I will use VS Code:
+
+First we need to connect our Medplum account.  Open "src/index.tsx".  Add a couple Medplum imports:
+
+```tsx
+import { MedplumClient } from '@medplum/core';
+import { MedplumProvider } from '@medplum/ui';
+```
+
+Add the Medplum client using the Client Application ID from before:
+
+```tsx
+const medplum = new MedplumClient({
+  clientId: '46705d5a-d0db-41b1-ae31-a8f5abe6135b'
+});
+```
+
+And add the MedplumProvider to the app:
+
+```tsx
+  <React.StrictMode>
+    <MedplumProvider medplum={medplum}>
+      <App />
+    </MedplumProvider>
+  </React.StrictMode>
+```
+
+When you're done, src/index.tsx should look like this:
+
+```tsx
+import { MedplumClient } from '@medplum/core';
+import { MedplumProvider } from '@medplum/ui';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+import reportWebVitals from './reportWebVitals';
+
+const medplum = new MedplumClient({
+  clientId: '46705d5a-d0db-41b1-ae31-a8f5abe6135b'
```

Comment:
There are a few places where I added dummy UUID's.  A couple options:

1. Leave as-is
2. Replace with placeholders like "YOUR_CLIENT_ID_HERE"
3. Add a `.env` file.  Extra complexity, but the "correct" way to do it

Reply:
I think #2 is probably best